src/mcpbr/benchmarks/supermodel/benchmark.py

```
@@ -0,0 +1,762 @@
+                "entryPoints": slim_entry_points,
+            }
+            index_path = Path(host_workdir) / self._endpoint.analysis_filename
+            index_path.write_text(json.dumps(index_data, indent=2))
+
+            logger.info(
+                f"Placed analysis for {instance_id}: {total} candidates "
+                f"in {len(chunk_refs)} chunks, {len(slim_entry_points)} entry points "
+                f"(filtered: {type_filtered} types, {reason_filtered} file-never-imported, "
+                f"{ep_filtered} entry points)"
+            )
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)